src/core/collection.ml

```diff
@@ -0,0 +1,16 @@
+
+module type VectorLike = sig 
+    type 'a t 
+
+    val mem: 'a Equality.t -> 'a t -> 'a -> bool 
```

Comment:
I've used this recently, and struggled to find this, so I _should_ be able to remember what this does. But even seeing the name and type signature I still could not. Lol, that's how bad this name is. `contains` is a nice name, though javascript has named it `includes` for some silly reason.